Rakefile

```diff
@@ -1,167 +1,127 @@
-    unless $ios_success
-      puts "** Build failed"
-      exit(-1)
-    end
-    $ios_success = system("xcodebuild -workspace ./ReplicationAcceptance/ReplicationAcceptance.xcworkspace -scheme 'RA_Tests' -destination 'platform=iOS Simulator,OS=latest,name=iPhone 4S' test")
-    puts "\033[0;31m! iOS unit tests failed" unless $ios_success
-    if $ios_success
-        puts "** All tests executed successfully"
-    else
-        exit(-1)
-    end
+  test(REPLICATION_ACCEPTANCE_WS, REPLICATION_ACCEPTANCE_IOS, IOS_DEST)
 end
 
 desc "Run the replication acceptance tests for OS X with encrypted datastores"
 task :encryptionreplicationacceptanceosx do
-    # build using xcpretty as otherwise it's very verbose when running tests
-    $osx_success = system("xcodebuild -workspace ./ReplicationAcceptance/ReplicationAcceptance.xcworkspace -scheme 'RA_EncryptionTests_OSX' -destination 'platform=OS X' build | xcpretty; exit ${PIPESTATUS[0]}")
-    unless $osx_success
-      puts "** Build failed"
-      exit(-1)
-    end
-    $osx_success = system("xcodebuild -workspace ./ReplicationAcceptance/ReplicationAcceptance.xcworkspace -scheme 'RA_EncryptionTests_OSX' -destination 'platform=OS X' test")
-    puts "\033[0;31m! OS X unit tests failed" unless $osx_success
-    if $osx_success
-        puts "** All tests executed successfully"
-    else
-        exit(-1)
-    end
+  test(REPLICATION_ACCEPTANCE_WS, REPLICATION_ACCEPTANCE_ENCRYPTED_OSX, OSX_DEST)
 end
 
 desc "Run the replication acceptance tests for iOS with encrypted datastores"
 task :encryptionreplicationacceptanceios do
-    # build using xcpretty as otherwise it's very verbose when running tests
-    $ios_success = system("xcodebuild -workspace ./ReplicationAcceptance/ReplicationAcceptance.xcworkspace -scheme 'RA_EncryptionTests' -destination 'platform=iOS Simulator,OS=latest,name=iPhone 4S' build | xcpretty; exit ${PIPESTATUS[0]}")
-    unless $ios_success
-      puts "** Build failed"
-      exit(-1)
-    end
-    $ios_success = system("xcodebuild -workspace ./ReplicationAcceptance/ReplicationAcceptance.xcworkspace -scheme 'RA_EncryptionTests' -destination 'platform=iOS Simulator,OS=latest,name=iPhone 4S' test")
-    puts "\033[0;31m! iOS unit tests failed" unless $ios_success
-    if $ios_success
-        puts "** All tests executed successfully"
-    else
-        exit(-1)
-    end
+  test(REPLICATION_ACCEPTANCE_WS, REPLICATION_ACCEPTANCE_ENCRYPTED_IOS, IOS_DEST)
 end
 
-desc "pod update all included projects"
-task :podupdate => [:podupdatetests] do
-  sh "for i in ReplicationAcceptance Project\ndo\ncd $i ; pod update ; cd ..\ndone"
-end
+#
+#  Update docs
+#
 
 desc "Build docs and install to Xcode"
 task :docs do
   system("appledoc --keep-intermediate-files --project-name CDTDatastore --project-company Cloudant -o build/docs --company-id com.cloudant -i Classes/vendor -i Classes/common/touchdb Classes/")
 end
+
+#
+#  Helper methods and defines
+#
+
+# Workspaces
```

Comment:
Can we put these 'constants' at the top of the file? I think they make more sense there